tests/python/unittest/test_tir_transform_lower_tvm_builtin.py

```diff
@@ -172,6 +172,47 @@ def build_tir():
     tvm.testing.assert_allclose(a.numpy(), expected_value)
 
 
+def test_lower_overflow_int32():
+    @T.prim_func
+    def variance4(
```

Comment:
Can we only keep the minimal reproducible parts?

Reply:
Thanks, have updated to the minimal version.